Source/WebKit/GPUProcess/media/RemoteVideoFrameObjectHeap.cpp

```diff
@@ -56,7 +56,8 @@ static WorkQueue& remoteVideoFrameObjectHeapQueue()
 Ref<RemoteVideoFrameObjectHeap> RemoteVideoFrameObjectHeap::create(Ref<IPC::Connection>&& connection)
 {
     auto heap = adoptRef(*new RemoteVideoFrameObjectHeap(WTFMove(connection)));
-    heap->m_connection->addWorkQueueMessageReceiver(Messages::RemoteVideoFrameObjectHeap::messageReceiverName(), remoteVideoFrameObjectHeapQueue(), heap);
+    Ref workQueue { remoteVideoFrameObjectHeapQueue() };
```

Comment:
We should not introduce a local variable like this.

Reply:
Fixed!